.github/workflows/tests_gpu_cuda.yml

```diff
@@ -315,7 +315,7 @@ jobs:
           OMP_PROC_BIND: false
         run: |
           cd main/
-          PL_DEVICE=lightning.qubit python -m pytest tests/ $COVERAGE_FLAGS
+          PL_DEVICE=lightning.qubit python -m pytest tests/ -k "not test_native_mcm" $COVERAGE_FLAGS
```

Comment:
`test_native_mcm` is expensive, we only need to run it once in `tests_linux.yml`.